spec/features/comments_feature_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+feature 'Comments' do
+  # As a user
+  # So that I can share my thoughts on a picture
+  # I would like to write a comment on a picture
+
+  context 'when signed in' do
+    before(:each) do
+      user = FactoryGirl.create(:user, :first)
+      sign_in(user.email, user.password)
+    end
+
+    scenario 'a user can post a comment' do
+      post_picture
+      click_link 'Add a comment'
+      fill_in 'Comment', with: 'A nice comment'
+      click_button 'Comment'
+      expect(page).to have_content 'A nice comment'
+    end
+  end
+
+  context 'when not signed in' do
+    scenario 'comments cannot be posted' do
+      picture = FactoryGirl.create(:picture, user: FactoryGirl.create(:user, :first))
```

Comment:
Metrics/LineLength: Line is too long. [85/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)